DEVELOPING.rst

```diff
@@ -230,7 +230,7 @@ master
   Where the bleeding-edge developments happen.
 
 qa
-  What is installed on the `pre-production server <http://opendataqa.cern.ch>`_.
+  What is installed on the `pre-production server <http://opendata-qa.cern.ch>`_.
```

Comment:
Can you please also:

- pin hadolint to `v1.18.2` in order to remove the CI error;
- add link to `opendata-dev.cern.ch` for the `master` branch?

Reply:
E.g. instead of `Where the bleeding-edge developments happen.` you can put 

```
What is installed on the `development server <http://opendata-dev.cern.ch>`_.
```